### DISCUSSION TITLE
Issue with Multiple Cursors

### DISCUSSION THREAD

**Discussion by thecoolwinter** [krzyzanowskim/STTextView#17] [Questions]
It looks like there's an issue with the recent changes adding multiple cursor support. When an edit is applied, all cursors but the top left-most one are lost.

https://user-images.githubusercontent.com/35942988/234064618-d1e859f5-b35f-4c8f-a6aa-cc12ca2a8b1c.mov

> STTextView version: 0.5.3
> Using the packaged demo project

**Comment by krzyzanowskim** [comment]
It's actually [partially fixed](https://github.com/krzyzanowskim/STTextView/blob/fb622359243e303f7c76bd1a8c9f5de1f6d3df5b/Sources/STTextView/STTextContentStorage.swift#L44). Unfortunately Text Kit 2 is not ready for multiple insertion points and gets in a way randomly. It needs a bigger workaround but I didn't want to hold it back.
